includes/class-wc-calypso-bridge-fse.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Site Editing tweaks and improvements.
+ *
+ * @package WC_Calypso_Bridge/Classes
+ * @since   x.x.x
+ * @version x.x.x
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC Calypso Bridge FSE class
+ */
+class WC_Calypso_Bridge_FSE {
+
+	/**
+	 * Class instance.
+	 *
+	 * @var WC_Calypso_Bridge_FSE instance
+	 */
+	protected static $instance = false;
+
+	/**
+	 * Get class instance.
+	 */
+	public static function get_instance() {
+		if ( ! self::$instance ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Constructor.
+	 */
+	private function __construct() {
+
+		// Only in Ecommerce.
+		if ( ! wc_calypso_bridge_has_ecommerce_features() ) {
+			return;
+		}
+
+		if ( is_admin() ) {
+
+			// Change "Blog Home" template name in the Site Editor to something more suitable for Woo Express.
+			add_filter(
+				'default_template_types',
+				function( $arg ) {
+
+					if ( isset( $arg[ 'home' ][ 'title' ] ) && _x( 'Blog Home', 'Template name' ) === $arg[ 'home' ][ 'title' ] ) {
+						$arg[ 'home' ][ 'title' ] = __( 'Home' );
```

Comment:
Is it safe to omit the translation domain here? Should it be `gutenberg`?

Reply:
It's ok to use `__( 'Home' )` , as it's the default WP translation